CassandraDB/setup.js

```diff
@@ -0,0 +1,56 @@
+const cassie = require('./index.js');
+
+let dropItems = `DROP TABLE IF EXISTS mykea.items;`
+let dropReviews = `DROP TABLE IF EXISTS mykea.reviews;`
+
+let createItems = `CREATE TABLE IF NOT EXISTS items (
+  ID int,
+  TITLE text,
+  DESCRIPTION text,
+  "originalPrice" int,
+  "salePrice" int,
+  COLORS text,
+  SIZES text,
+  LIKED boolean,
+  "inStock" int,
+  PRIMARY KEY (ID)
+);`
+
+let createReviews = `CREATE TABLE IF NOT EXISTS reviews (
+  ID int,
+  "easeOfAssembly" int,
+  "valueForMoney" int,
+  "productQuality" int,
+  APPEARANCE int,
+  "worksAsExpected" int,
+  "overallRating" int,
+  "createdAt" text,
+  "iRecommendThisProduct" boolean,
+  HEADER text,
+  BODY text,
+  PRIMARY KEY (ID, "createdAt")
+);`
+
+// paste these lines into cqlsh after creating the tables
+// COPY mykea.items (id, title, description, "originalPrice", "salePrice", colors, sizes, liked, "inStock") FROM 'DataGeneration/items.csv' WITH DELIMITER=',' AND HEADER=TRUE ;
+// COPY mykea.reviews (id, "easeOfAssembly", "valueForMoney", "productQuality", appearance, "worksAsExpected", "overallRating", "createdAt", "iRecommendThisProduct", header, body) FROM 'DataGeneration/reviews.csv' WITH DELIMITER=',' AND HEADER=TRUE ;
+
+
+cassie.execute(dropItems)

```

Comment:
Stu this looks great, I like how you took advantage of promises! Wondering after your create items you have *IF NOT EXISTS* but since you're dropping them at the beginning, that's actually not needed correct? I'm assuming you included those just in case, and I'm not suggesting you got back and remove it, I'm just curious if there's something weird with Cassandra that I should be aware of.

Reply:
You're absolutely right that if I always drop them before I create them, if not exists isn't really doing a lot. It's mostly still in there from a previous version of doing things. I definitely could remove it if I have time to refactor later. Thanks for pointing it out.